diff --git a/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js b/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
index ff7be71..b32df59 100644
--- a/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
+++ b/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
@@ -108,12 +108,14 @@ require(['gitbook', 'jquery'], function(gitbook, $) {
         // Remove currently applied color theme
         if (fontState.theme !== 0)
             $book.removeClass('color-theme-'+fontState.theme);
+            $('body').removeClass('color-theme-'+fontState.theme);
 
         // Set new color theme
         var themeId = getThemeId(configName);
         fontState.theme = themeId;
         if (fontState.theme !== 0)
             $book.addClass('color-theme-'+fontState.theme);
+            $('body').addClass('color-theme-'+fontState.theme);
 
         saveFontSettings();
     }
@@ -153,6 +155,7 @@ require(['gitbook', 'jquery'], function(gitbook, $) {
         if(fontState.theme !== 0) {
             $book[0].className = $book[0].className.replace(/\bcolor-theme-\S+/g, '');
             $book.addClass('color-theme-'+fontState.theme);
+            $('body').addClass('color-theme-'+fontState.theme);
         }
     }
 
diff --git a/node_modules/gitbook-plugin-fontsettings/assets/website.css b/node_modules/gitbook-plugin-fontsettings/assets/website.css
index 26591fe..d162711 100644
--- a/node_modules/gitbook-plugin-fontsettings/assets/website.css
+++ b/node_modules/gitbook-plugin-fontsettings/assets/website.css
@@ -44,6 +44,9 @@
   line-height: 30px;
   font-size: 1em;
 }
+body.color-theme-1 {
+  background: #f3eacb;
+}
 .book.color-theme-1 .book-body {
   color: #704214;
   background: #f3eacb;
@@ -51,6 +54,9 @@
 .book.color-theme-1 .book-body .page-wrapper .page-inner section {
   background: #f3eacb;
 }
+body.color-theme-2 {
+  background: #1c1f2b;
+}
 .book.color-theme-2 .book-body {
   color: #bdcadb;
   background: #1c1f2b;
