diff --git a/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js b/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
index ff7be71..fb65802 100644
--- a/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
+++ b/node_modules/gitbook-plugin-fontsettings/assets/fontsettings.js
@@ -1,8 +1,7 @@
 require(['gitbook', 'jquery'], function(gitbook, $) {
     // Configuration
     var MAX_SIZE       = 4,
-        MIN_SIZE       = 0,
-        BUTTON_ID;
+        MIN_SIZE       = 0;
 
     // Current fontsettings state
     var fontState;
@@ -64,6 +63,10 @@ require(['gitbook', 'jquery'], function(gitbook, $) {
 
     // Save current font settings
     function saveFontSettings() {
+        // Fixed font size and family
+        fontState.size = 2;
+        fontState.family = 1;
+
         gitbook.storage.set('fontState', fontState);
         update();
     }
@@ -106,14 +109,24 @@ require(['gitbook', 'jquery'], function(gitbook, $) {
         var $book = gitbook.state.$book;
 
         // Remove currently applied color theme
-        if (fontState.theme !== 0)
+        if (fontState.theme !== 0) {
             $book.removeClass('color-theme-'+fontState.theme);
+            $('body').removeClass('color-theme-'+fontState.theme);
+            $('.theme-setting i').removeClass('fa-sun-o');
+        } else {
+            $('.theme-setting i').removeClass('fa-moon-o');
+        }
 
         // Set new color theme
         var themeId = getThemeId(configName);
         fontState.theme = themeId;
-        if (fontState.theme !== 0)
+        if (fontState.theme !== 0) {
             $book.addClass('color-theme-'+fontState.theme);
+            $('body').addClass('color-theme-'+fontState.theme);
+            $('.theme-setting i').addClass('fa-sun-o');
+        } else {
+            $('.theme-setting i').addClass('fa-moon-o');
+        }
     
         saveFontSettings();
     }
@@ -172,44 +185,17 @@ require(['gitbook', 'jquery'], function(gitbook, $) {
     }
 
     function updateButtons() {
-        // Remove existing fontsettings buttons
-        if (!!BUTTON_ID) {
-            gitbook.toolbar.removeButton(BUTTON_ID);
-        }
-
         // Create buttons in toolbar
-        BUTTON_ID = gitbook.toolbar.createButton({
-            icon: 'fa fa-font',
-            label: 'Font Settings',
-            className: 'font-settings',
-            dropdown: [
-                [
-                    {
-                        text: 'A',
-                        className: 'font-reduce',
-                        onClick: reduceFontSize
-                    },
-                    {
-                        text: 'A',
-                        className: 'font-enlarge',
-                        onClick: enlargeFontSize
-                    }
-                ],
-                $.map(FAMILIES, function(family) {
-                    family.onClick = function(e) {
-                        return changeFontFamily(family.config, e);
-                    };
-
-                    return family;
-                }),
-                $.map(THEMES, function(theme) {
-                    theme.onClick = function(e) {
-                        return changeColorTheme(theme.config, e);
-                    };
-
-                    return theme;
-                })
-            ]
+        gitbook.toolbar.createButton({
+            icon: 'fa fa-moon-o',
+            position: 'right',
+            className: 'theme-setting',
+            onClick: function(e) {
+                if (fontState.theme !== 0)
+                    changeColorTheme("wight", e);
+                else
+                    changeColorTheme("night", e);
+            }
         });
     }
 
diff --git a/node_modules/gitbook-plugin-fontsettings/assets/website.css b/node_modules/gitbook-plugin-fontsettings/assets/website.css
index 26591fe..d162711 100644
--- a/node_modules/gitbook-plugin-fontsettings/assets/website.css
+++ b/node_modules/gitbook-plugin-fontsettings/assets/website.css
@@ -44,6 +44,9 @@
   line-height: 30px;
   font-size: 1em;
 }
+body.color-theme-1 {
+  background: #f3eacb;
+}
 .book.color-theme-1 .book-body {
   color: #704214;
   background: #f3eacb;
@@ -51,6 +54,9 @@
 .book.color-theme-1 .book-body .page-wrapper .page-inner section {
   background: #f3eacb;
 }
+body.color-theme-2 {
+  background: #1c1f2b;
+}
 .book.color-theme-2 .book-body {
   color: #bdcadb;
   background: #1c1f2b;
