diff --git a/node_modules/gitbook-plugin-chapter-fold/book/chapter-fold.js b/node_modules/gitbook-plugin-chapter-fold/book/chapter-fold.js
index a80a921..34f8b3b 100644
--- a/node_modules/gitbook-plugin-chapter-fold/book/chapter-fold.js
+++ b/node_modules/gitbook-plugin-chapter-fold/book/chapter-fold.js
@@ -6,7 +6,7 @@ require(['gitbook', 'jQuery'], function(gitbook, $) {
       LS_NAMESPACE = 'expChapters';
   var init = function () {
     // adding the trigger element to each ARTICLES parent and binding the event
-    var chapterLink = $(ARTICLES).parent(CHAPTER).children('a');
+    var chapterLink = $(ARTICLES).parent(CHAPTER).children('span');
     chapterLink.append($(TRIGGER_TEMPLATE));
     chapterLink.on('click', function (e) {
       e.preventDefault();
@@ -62,6 +62,16 @@ require(['gitbook', 'jQuery'], function(gitbook, $) {
     }
   }
   gitbook.events.bind('page.change', function() {
+    if (gitbook.storage.get('fontState')) {
+      if (gitbook.storage.get('fontState')['theme'] !== 0) {
+        $('.theme-setting i').removeClass('fa-moon-o');
+        $('.theme-setting i').addClass('fa-sun-o');
+      } else {
+        $('.theme-setting i').removeClass('fa-sun-o');
+        $('.theme-setting i').addClass('fa-moon-o');
+      }
+    }
+
     init()
   }); 
 });
